modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -0,0 +1,170 @@
++
+[TIP]
+====
+[source,terminal]
+----
+$ oc apply -f <secret_file>.yaml
+----
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
+====
+
+include::snippets/distr-tracing-tempo-required-secret-parameters-and-secret-example.adoc[]
+
+. Use `oc create namespace` or `oc apply` to create a namespace for the *TempoStack* instance that you will create in the next step.
++
+[TIP]
+====
+[source,terminal]
+----
+$ oc create namespace <namespace_value_of_your_choice>
+----
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  labels:
+    kubernetes.io/metadata.name: <namespace_value_of_your_choice>
+    openshift.io/cluster-monitoring: "true"
+  name: <namespace_value_of_your_choice>
+EOF
+----
+====
+
+. Create a *TempoStack* instance in the namespace that you created for the *TempoStack* instance in the previous step.
++
+NOTE: You can create multiple *TempoStack* instances in separate namespaces on the same cluster.
++
+.. Customize the `TempoStack` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+  namespace: <namespace_from_previous_step>
+spec:
+  storageSize: 1Gi
+  storage:
+      secret:
+          name: <secret-name> <1>
+          type: <secret-provider> <2>
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+<1> The value of `name` in the `metadata` of the Secret.
+<2> The accepted values are `azure` for Azure Blob Storage, `gcs` for Google Cloud Storage, and `s3` for Amazon S3 or MinIO or Red Hat OpenShift Data Foundation.
++
+.TempoStack CR for AWS S3 and MinIO storage
+====
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+  namespace: tempostacknamespace
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      name: minio-test
+      type: s3
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+The stack deployed in this example is configured to receive Jaeger Thrift over HTTP and OpenTelemetry Protocol (OTLP), which permits visualizing the data with the Jaeger UI.
+====
+
+.. Apply the customized CR.
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<TempoStack_custom_resource>
+EOF
+----
+
+.WIP:
+
+. Command(s) to verify successful deployment of the TempoStack instance
++
+For example:
+.. A command that prints output to verify that pods are created on the namespace.
+.. A command that enables the CLI user to watch for the stack to stabilize.
+
+. If applicable, a command to open any UI console
+. Any other commands?
```

Comment:
@rubenvp8510 and @IshwarKanse, when you can find the time, could you come up with suggested changes for these lines here?